packages/ui/src/ui-component/lists/FlowListView.jsx

```diff
@@ -0,0 +1,61 @@
+import PropTypes from 'prop-types'
+import { Box, Grid, Skeleton, Typography } from '@mui/material'
+import ItemCard from '@/ui-component/cards/ItemCard'
+
+const FlowListView = ({ data, images, nodeTypes, isLoading, updateFlowsApi, setError, type, onItemClick }) => {
+    return (
+        <Box sx={{ display: 'flex', flexDirection: 'column', gap: 3 }}>
+            <Grid container spacing={3}>
+                {isLoading ? (
+                    Array.from({ length: 6 }).map((_, index) => (
+                        <Grid item xs={4} sm={4} md={4} key={index}>
+                            <Skeleton variant='rectangular' height={200} />
+                        </Grid>
+                    ))
+                ) : data.length > 0 ? (
+                    data.map(
+                        (item) =>
+                            item && (
+                                <Grid item xs={4} sm={4} md={4} key={item.id || item.templateId}>
```

Comment:
I recommend making the key more unique if possible.  It may be unique enough though?